torch/_inductor/compile_fx.py

```diff
@@ -501,12 +505,12 @@ def fx_codegen_and_compile(
     # on node.meta["val"]. if in the future we rely on these being
     # correct we will need to fix.
 
-    with V.set_fake_mode(fake_mode):
+    with V.set_fake_mode(fake_mode):  # type: ignore[call-arg]
```

Comment:
If it's not too hard to make this type check, that would be nice since it's a common idiom.

Reply:
Good catch. Removing the ```ignore``` doesn't trigger any mypy failure. Forgot why I added it here...